### DISCUSSION TITLE
Suggestion : type tuple

### DISCUSSION THREAD

**Discussion by math1992** [json-schema-org/community#629] [General]
After many years using the draft-07, my employer decided to update to draft 2020-12. One confusing thing that truly strike about this draft version is the introduction of the prefixItems and items for tuple, which is confusing, mainly when maintaining the schema.

My suggestion is rather than introducing new keywords, why not simply introduce a new type **tuple**?

For one, the true intention is clear, an array containing different type, for two, the word tuple is recognized in the IT community and would not cause any confusion if introduced.

**Comment by gregsdennis** [comment]
We had considered something "tuple"-related as a name for `prefixItems`, but the argument against that was that the usefulness of the field goes beyond only tuples.

The intent for the keyword is that you could have any ordered set of different types.  While tuples are a common use case, they're not the only one, and so naming it "tuple" isn't general enough.
